pkg/controller/che/che_controller.go

```diff
@@ -771,8 +771,21 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 		}
 	}
 
+	trustStoreCMResourceVersion := ""
+	if instance.Spec.Server.ServerTrustStoreConfigMapName != "" {
+		trustStoreConfigMap, _ := deploy.GetClusterConfigMap(instance.Spec.Server.ServerTrustStoreConfigMapName, instance.Namespace, clusterAPI.Client)
+		if trustStoreConfigMap != nil {
+			// trustStoreConfigMap might be created by user, to detect changes we have to add the owner
```

Comment:
Why ? Isn't the main method run regularly anyway ?
Not sure what the requirement for an owner will mean when we select all the config maps that have a given label.